<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shapeColorPickClass</class>
 <widget class="QDialog" name="shapeColorPickClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shapeColorPick</string>
  </property>
  <widget class="QComboBox" name="shpColorTypeCmb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a color!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shpColorDialogBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>34</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shpColorSelectBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter line width!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="shpColorWidthTxt">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
